src/components/Image/src/Image.vue

```diff
@@ -252,5 +284,15 @@ export default {
 		border-top-left-radius: var(--image-height);
 		border-top-right-radius: var(--image-height);
 	}
+
+	&.shape_hexagon {
+		-webkit-clip-path: url(#mImageHexPath);
+		clip-path: url(#mImageHexPath);
+	}
+}
+
+.ImageClipPaths {
+	width: 0;
+	height: 0;
```

Comment:
Could we add an `overflow: hidden` here just in case the svg inherits anything that would impact it's dimensions or visibility from context?

Reply:
Just added